services/director-v2/requirements/_test.in

```diff
@@ -17,6 +17,7 @@ async-asgi-testclient # replacement for fastapi.testclient.TestClient [see b) be
 codecov
 coveralls
 dask-gateway-server[local]
+dask[distributed,diagnostics]
```

Comment:
this is not already in ``_base.txt``? ie. with ``--requirement ../../../services/dask-sidecar/requirements/_dask-distributed.txt`` ?